### PR TITLE
Updated Hungarian Language

### PR DESCRIPTION
**Pull request by AntoszHUN**
I updated the Hungarian language files.
Translation of the newly added elements into Hungarian.

These were done in the language_hu.h and language_hu.ini files.

Grammar errors may still occur.
Their improvement is continuous.

Made By: AntoszHUN

### CODE REVIEW

File: Copy to SD Card root directory to update/Language Packs/language_hu.ini [hunk 1]
```diff
@@ -1,321 +1,321 @@
...
+label_persistent_info:Perzisztens állapotinfó
+label_file_list_mode:Fájlnéző listamód
+label_ack_notification:ACK Értesítési stílus
+label_default_mode:Alapértelmezett mód
+label_serial_always_on:Soros mindig BE
+label_marlin_show_title:Marlin mód cím támogatás
+label_marlin_fullscreen:Nagyképernyős Marlin mód
+label_marlin_type:Marlin mód típusa
+label_move_speed:Mozgási seb. (X-Y-Z)
+label_auto_load_leveling:Automentés terhel. szint.
```
Comment (kisslorand) [inline Copy to SD Card root directory to update/Language Packs/language_hu.ini:23]: Save = mentés
Load = betöltés

File: Copy to SD Card root directory to update/Language Packs/language_hu.ini [hunk 2]
```diff
@@ -1,321 +1,321 @@
...
+label_read_tftsd_error:TFT-SD olvasási hiba!
+label_tftsd_inserted:SD-Kártya behelyezve!
+label_tftsd_removed:SD-Kártya eltávolítva!
+label_u_disk:USB Tár
+label_read_u_disk_error:USB olvasási hiba!
+label_u_disk_inserted:USB behelyezve!
+label_u_disk_removed:USB eltávolítva!
+label_onboardsd:Lap-SD
+label_read_onboardsd_error:Alaplap-SD olvasási hiba!
+label_filament_runout:Szál kifutás!
```
Comment (kisslorand) [inline Copy to SD Card root directory to update/Language Packs/language_hu.ini:154]: run out = elfogy

**Comment by digant73** [comment]
hi, please in soruce files do not translate also the comments. They must remain in english language.

**Review by AntoszHUN — COMMENTED** [review]
> hi, please in soruce files do not translate also the comments. They must remain in english language.

I tested it. There was no problem with the translation.

**Comment by digant73** [comment]
yes, it's not a problem or a bug. The comments in the source code must not be translated (according to style guide, english is the only language that must be used).

**Comment by AntoszHUN** [comment]
> yes, it's not a problem or a bug. The comments in the source code must not be translated (according to style guide, english is the only language that must be used).

All right. I'll write back in the next fix.

**Comment by digant73** [comment]
no problem. already done. Just to avoid to create a PR just for this very minor cleanup I provided the change in an existing PR

**Comment by AntoszHUN** [comment]
> no problem. already done. Just to avoid to create a PR just for this very minor cleanup I provided the change in an existing PR 1598

All right. Thanks!

**Pull request merged**